downstream/modules/eda/ref-eda-controller-install-builder.adoc

```diff
@@ -0,0 +1,11 @@
+[id="eda-controller-install-builder"]
```

Comment:
To reviewers: This is a new file that duplicates the same "Install ansible-builder" topic found in the controller docs.